resources/services/shield/protections.go

```diff
@@ -0,0 +1,122 @@
+package shield
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/shield"
+	"github.com/aws/aws-sdk-go-v2/service/shield/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource protections --config gen.hcl --output .
+func Protections() *schema.Table {
+	return &schema.Table{
+		Name:          "aws_shield_protections",
+		Description:   "An object that represents a resource that is under DDoS protection.",
+		Resolver:      fetchShieldProtections,
+		Multiplex:     client.AccountMultiplex,
+		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:  client.DeleteAccountFilter,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
+		IgnoreInTests: true,
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "tags",
+				Description: "The AWS tags of the resource.",
+				Type:        schema.TypeJSON,
+				Resolver:    ResolveShieldProtectionTags,
+			},
+			{
+				Name:        "application_layer_automatic_response_configuration_status",
+				Description: "Indicates whether automatic application layer DDoS mitigation is enabled for the protection.  This member is required.",
```

Comment:
remover member is required from descriptions.

Maybe shorten to application_layer_protection_status

Reply:
cc @bbernays